docs/developer-guide/using-the-polyfill-service.md

```diff
@@ -7,9 +7,18 @@ site_section: developer-guide
 
 ---
 
-Stub.
-This page should:
+Not all browsers are equal. As new standards and features are added to browsers, older ones (eg ie8) are left behind. "Polyfilling" is a technique where older browsers are served JavaScript which upgrades their feature-set. This frees up developers to write modern JavaScript without having to spend time getting it working on older browsers.
 
-1. explain polyfilling
-1. explain the benefits of using the polyfill service
-1. explain how to use the polyfill service
+Ordinarily, developers have to identify and manage the polyfills they need. This is where Polyfill.io comes in. Polyfill.io uses user-agent sniffing to return only the polyfills that the requesting browser needs. Polyfill.io is built and maintained by a community of developers led by Origami.
+
+All documentation for Polyfill.io is available at (Polyfill.io)[http://polyfill.io].
+
+
+## Using Polyfill.io with other polyfills.
+Polyfilling things twice will give unpredictable results and is likely to lead to bugs in the browsers that use those polyfills. To avoid conflicts, where possible, you should only use polyfills from a single source.
+
+If you want to use a polyfill that isn't provided by Polyfill.io, you should not use the default set from Polyfill.io as this can lead to conflicts. These conflicts are especially common for ES6 features while Polyfill.io brings in polyfills for these new language features.
+
+
+## Polyfill.io and the Build Service.
+Origami Build Tools and the Build Service both include some polyfills from core-js which conflict with the Polyfill.io's default set. Until we remove these conflicting polyfills from the Build Service (when Polyfill.io can provide the same coverage for ES6 as offered in core-js) you should use `?excludes=Symbol,Symbol.iterator,Symbol.species,Map,Set` to prevent conflicts.
```

Comment:
We should mention that we would also be hoping to remove the polyfills from OBT.